webapp/components/Notification/Notification.spec.js

```diff
@@ -198,4 +199,71 @@ describe('Notification', () => {
       })
     })
   })
+
+  describe('given a notification about the user has filed a report', () => {
+    beforeEach(() => {
+      propsData.notification = {
+        reason: 'filed_report_on_resource',
+        from: {
+          __typename: 'Report',
+          id: 'reportOnUser',
+          filed: [
+            {
+              reasonCategory: 'discrimination_etc',
+              reasonDescription: 'This user is harassing me with bigoted remarks!',
+              reportedResource: {
+                __typename: 'User',
+                id: 'badWomen',
+                slug: 'mrs.-badwomen',
+                name: 'Mrs. Badwomen',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:44:49Z" title="Thursday, January 23rd 2020, 7:44:49 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -198,4 +199,71 @@ describe('Notification', () => {
       })
     })
   })
+
+  describe('given a notification about the user has filed a report', () => {
+    beforeEach(() => {
+      propsData.notification = {
+        reason: 'filed_report_on_resource',
+        from: {
+          __typename: 'Report',
+          id: 'reportOnUser',
+          filed: [
+            {
+              reasonCategory: 'discrimination_etc',
+              reasonDescription: 'This user is harassing me with bigoted remarks!',
+              reportedResource: {
+                __typename: 'User',
+                id: 'badWomen',
+                slug: 'mrs.-badwomen',
+                name: 'Mrs. Badwomen',
```


I guess you mean `badWoman` as it's a singular user... I think it must be tough to remember that... not that it really matters...
maybe `harrassing-user` ... but don't really care

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T09:31:05Z" title="Friday, January 24th 2020, 10:31:05 am +01:00">Jan 24, 2020</time>_

---


I thought it’s funny!?